reason-react-native/src/elements/VirtualizedListElement.re

```diff
@@ -0,0 +1,14 @@
+type element;
+type ref = React.Ref.t(Js.nullable(element));
+
+include VirtualizedListMethods.Make({
+  type t = element;
+});
+
+include ScrollViewMethods.Make({
+  type t = element;
+});
+
+include NativeMethods.Make({
+  type t = element;
+});
```

Comment:
Shouldn't we include in the reverse order? In case something more specific overwrite a more generic method?
Not sure if that's possible tho

Reply:
Overriding is not possible.